src/homekit.cs

```diff
@@ -1671,12 +1683,65 @@ interface HMNumberRange {
 		NSNumber Max { get; }
 	}
 
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAccessoryOwnershipToken
+	{
+		[Export("initWithData:")]
+		IntPtr Constructor(NSData data);
+	}
+
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAddAccessoryRequest
+	{
+		[Export ("home")] 
+		HMHome Home { get; [NotImplemented] set; }
+		
+		[Export ("accessoryCategory")]
+		HMAccessoryCategory AccessoryCategory { get; [NotImplemented] set; }
+		
+		[Export ("accessoryName")]
+		NSString AccessoryName { get; [NotImplemented] set; }
+		
+		[Export ("requiresSetupPayloadURL")]
+		bool RequiresSetupPayloadURL { get; [NotImplemented] set; }
+		
+		[Export ("requiresOwnershipToken")]
+		bool RequiresOwnershipToken { get; [NotImplemented] set; }
+
+		[NullAllowed, Export ("payloadWithOwnershipToken:")]
+		HMAccessorySetupPayload GetPayloadWithOwnershipToken (HMAccessoryOwnershipToken ownershipToken);
+		
+		[NullAllowed, Export ("payloadWithURL:ownershipToken:")]
+		HMAccessorySetupPayload GetPayloadWithUrlAndOwnershipToken (NSUrl setupPayloadUrl, HMAccessoryOwnershipToken ownershipToken);
+	}
+
+	[iOS (13,0), Watch (6,0), TV (13,0), NoMac]
+	[DisableDefaultCtor]
+	interface HMNetworkConfigurationProfile : HMAccessoryProfile {
+		[Export ("delegate"), ArgumentSemantic.Weak)]
+		[NullAllowed]
+		HMNetworkConfigurationProfileDelegate WeakDelegate { get; set; }
+		
+		[Export ("isNetworkAccessRestricted")]
```

Comment:
While what you did is correct, we like to preserve the information like in the headers so this is what we would do.

```suggestion
		[Export ("networkAccessRestricted")]
```